app/templates/sidebars/settings.hbs

```diff
@@ -1,18 +1,47 @@
-<nav class="layout-sidebar">
-  <ul class='nav sidebar-nav'>
+{{#each session.currentUser.organizations as |organization|}}
+  <ul class="nav sidebar-nav">
     <li class="title">
-      {{session.currentUser.name}} Settings
+      {{organization.name}} Settings
     </li>
-    {{#activating-item currentWhen='settings.profile' tagName='li'}}
-      {{link-to 'Profile' 'settings.profile'}}
-    {{/activating-item}}
+    {{#link-to 'organization.members' organization tagName="li"}}
+      {{link-to "Members" "organization.members" organization bubbles=false}}
+    {{/link-to}}
+    {{#link-to 'organization.roles' organization tagName="li"}}
+      {{link-to "Roles" "organization.roles" organization bubbles=false}}
+    {{/link-to}}
+    {{#link-to 'organization.environments' organization tagName="li"}}
```

Comment:
@rwjblue -- Is there a better way to do this?  We also have a `{{#activating-item}}` component that will activate for the current route path, but it doesn't consider resource IDs, so it won't work here when a user could potentially have multiple organizations.